### DISCUSSION TITLE
What is the point of the .container-md class in bootstrap?

### DISCUSSION THREAD

**Discussion by ibnteo** [twbs/bootstrap#38052] [Q&A]
It just seems that `.container` and `.container-md` work the same way. Redefined these classes as:
```
.container-md {
max-width: 720px;
}
```
and now the container is no larger than this size, it just makes more sense.

**Comment by denizgolbas** [comment]
.container-md is 100% wide to start until the md breakpoint is reached

**Reply by ibnteo** [reply]
But this is not being done now, .container-md is no different from .container

**Reply by mdo** [reply]
They are not the same. See the top near the top of the page at https://getbootstrap.com/docs/5.3/layout/containers/.

**Reply by ibnteo** [reply]
Yes, they are not the same, but do they make any sense?
Here in containers with a fixed maximum width just makes sense.

**Reply by mdo** [reply]
Yes, they make sense :). They do different things as mentioned, and there are unique use cases for both.
